tests/StrictObjects/SuggesterTest.phpt

```diff
@@ -55,14 +57,20 @@ class SuggesterTest extends \Tester\TestCase
 	}
 
 	/**
+	 * @param string|null $expected
```

Comment:
I think SlevomatCodingStandard.TypeHints.TypeHintDeclaration supersedes it with better behavior, I'd just remove it from 7.1 CS.